FastText/src/models/train_text_classifier.jl

```diff
@@ -0,0 +1,60 @@
+"""
+ULMFiT - Text Classifier
+
+This is wrapper around the LanguageMode struct. It has three fields:
+
+vocab           : contains the same vocabulary from the LanguageModel
+rnn_layers      : contains same DroppedEmebeddings, LSTM (AWD_LSTM) and VarDrop layers of LanguageModel except for last softmax layer
+linear_layers   : contains Chain of two Dense layers [PooledDense and Dense] with softmax layer
+
+To train create and instance and give it as first argument to 'train_classifier!' function
+"""
+mutable struct TextClassifier{A,F,Q}
+    vocab::A
+    rnn_layers::F
+    linear_layers::Q
+end
+
+function TextClassifier(lm::LanguageModel = LanguageModel(), clsfr_out_sz::Integer = 2, clsfr_hidden_sz::Integer = 50,
+    clsfr_hidden_drop::Float32 = 0.4f0
+)
+    return TextClassifier(
+        lm.vocab,
+        lm.layers[1:8],
+        Chain(
+            PooledDense(length(lm.layers[7].cell.h), clsfr_hidden_sz),
+            BatchNorm(clsfr_hidden_sz, relu),
+            Dropout(clsfr_hidden_drop),
+            Dense(clsfr_hidden_sz, clsfr_out_sz),
+            BatchNorm(clsfr_out_sz),
+            softmax
+        )
+    )
+end
+
+Flux.@functor TextClassifier
```

Comment:
```suggestion
Flux.@functor TextClassifier
Flux.trainable(m::TextClassifier) = (rnn_layers = m.rnn_layers, linear_layers = m.linear_layers)
```

Reply:
No luck! The problem still remains the same.